example_package/doc/abczad.tex

```diff
@@ -79,20 +79,21 @@
 
 \tSection{Testy ocen:}
 \ocen{
-  \testOcen{1ocen}{$\tc{n} = 69$, Założenie}
-  \testOcen{2ocen}{$\tc{n} = 420$, założenie 2}
+  \testOcen{1ocen}{$\tc{n} = 69$, Założenie jakieś}
+  \testOcen{2ocen}{$\tc{n} = 420$, założenie jakieś 2}
 }
 
 \tSection{Ocenianie}
 
 Zestaw testów dzieli się na następujące podzadania.
 Testy do każdego podzadania składają się z jednej lub większej liczby osobnych grup testów.
 
-% Alternatywnie \customOcenTable{|c|c|c|}{\tc{Nr} & \tc{Ograniczenia} & \tc{Punkty}}{reszta tabeli}
+% Alternatywnie \customOcenTable{|c|c|c|}{\tc{Nr} & \tc{Ograniczenia} & \tc{Punkty}}{reszta tabeli & & }
```

Comment:
Co tu robią te &&? 
Jeżeli już mielibyśmy jakoś sugerować, jak się robi tabele w latexie, to trzebaby to porządniej opisać, bo to może wręcz wprowadzić w błąd.